src/containers/WelcomeContainer/WelcomeContainer.scss

```diff
@@ -6,7 +6,10 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  background-color: $red;
+  background: -webkit-linear-gradient(left top, $red, $purplesmooth); 
```

Comment:
we only need linear-gradient here because we have libraries that do all these polifills